libs/SalesforceSDKCore/SalesforceSDKCore/Classes/Network/Network/Action/CSFSalesforceAction.m

```diff
@@ -324,18 +296,37 @@ + (BOOL)isNetworkError:(nullable NSError *)error {
 }
 
 #pragma mark SFAuthenticationManagerDelegate
-
-- (void)userAccountManagerDidChangeCurrentUser:(NSNotification*)notification {
+- (void)userAccountManagerDidChangeUserDataNotification:(NSNotification*)notification {
     SFUserAccountManager *accountManager = (SFUserAccountManager*)notification.object;
```

Comment:
Is there any reason why the notification's object is the `SFUserAccountManager`? Wouldn't it make more sense for the object to be the user account that's being changed?  That way, the action can subscribe to the notification based on that object.  e.g.

```
[[NSNotificationCenter defaultCenter] addObserver:self
                                   selector:@selector(userAccountManagerDidChangeUserDataNotification:)
                                       name:SFUserAccountManagerDidChangeUserDataNotification
                                     object:self.enqueuedNetwork.account];
```

That way the actions will only receive notifications for changes to the account they care about.